clients/cobol-lsp-vscode-extension/src/services/BridgeForGit.ts

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright (c) 2024 Broadcom.
+ * The term "Broadcom" refers to Broadcom Inc. and/or its subsidiaries.
+ *
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *   Broadcom, Inc. - initial API and implementation
+ */
+import * as t from "io-ts";
+import { isLeft } from "fp-ts/Either";
+import { PathReporter } from "io-ts/PathReporter";
+import path = require("path");
```

Comment:
Please, prefer VS code APIs which are universal across all environments instead of node specific APIs.

Typical example `node:fs` => `vscode.workspace.fs` and `node:path` => `vscode.Uri`.